### PR TITLE
Create GCP Cloud Function wrapper for Address Standardization and Geocoding

### PR DESCRIPTION
**Pull request by BradySkylight**
### Description
This PR includes the following functionality:

- A HTTP Function to handle the address standardization and gecoding for the GCP Cloud functions utilizing the PHDI SDK functionality under the hood
- Unit tests
- Terraform coverage for the geocode_patients GCP Function
- Terraform coverage to support the storing/getting GH Secrets for SMARTY (Auth id, and Auth Token)
- Update the getting started scripts (Both Unix and Windows) to provide ability to auto add SMARTY auth id and token or instructions on how to manually add the SMARTY secrets to their GitHub repo
- Minor fixes to some other test cases to increase similarity between all our tests
- Updates to the Getting Started documentation

### CODE REVIEW

File: cloud-functions/http_geocode_patients/main.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+            os.environ.get("SMARTY_AUTH_ID"),
+            os.environ.get("SMARTY_AUTH_TOKEN"),
+        )
+
+        request_json = request.get_json(silent=False)
+        # Perform the address standardization and geocoding and
+        # store results in flask response
+        body_response = make_response(
+            status_code=200,
+            json_payload=geocode_patients(bundle=request_json, client=geocoder),
```
Comment (DanPaseltiner) [inline cloud-functions/http_geocode_patients/main.py:56]: I think we should pull `geocode_patients(bundle=request_json, client=geocoder)` out of the `make_response()` call and wrap it in a try/except block. Maybe the `get_smartystreets_client` should be included in this block as well. We just need to be able to handle problems on Smarty's end gracefully.

**Review by DanPaseltiner — CHANGES_REQUESTED** [review]
This look really great. Thank you for all the work updating documentation and both of the quick start scripts! The only thing that is blocking from my perspective is that we don't have error handling for our calls to Smarty.

File: cloud-functions/http_geocode_patients/main.py (hunk 1 above)
Comment (BradySkylight) [reply cloud-functions/http_geocode_patients/main.py]: Yup.  I just did this actually.  Good catch.

File: cloud-functions/http_geocode_patients/requirements.txt [hunk 2]
```diff
@@ -0,0 +1,5 @@
+Flask==2.2.2
+functions-framework==3.2.0
+phdi 
+phdi_cloud_function_utils @ git+https://github.com/CDCgov/phdi-google-cloud@main#subdirectory=cloud-functions/phdi_cloud_function_utils 
+pytest==7.1.2
```
Comment (DanPaseltiner) [inline cloud-functions/http_geocode_patients/requirements.txt:5]: I think we can remove pytest as it is only a dev requirement.

**Review by DanPaseltiner — APPROVED** [review]
LGTM!

**Pull request merged**